src/plus/webviews/graph/graphWebview.ts

```diff
@@ -1066,6 +1097,41 @@ export class GraphWebview extends WebviewBase<State> {
 		return this.container.storage.getWorkspace('graph:columns');
 	}
 
+	private getHiddenRefs(): Record<string, GraphHiddenRef> | undefined {
+		return this.container.storage.getWorkspace('graph:hiddenRefs');
+	}
+
+	private async getHiddenRefsById(
+		hiddenRefs: Record<string, GraphHiddenRef> | undefined
+	): Promise<GraphHiddenRefs | undefined> {
+		if (hiddenRefs == null) return undefined;
+
+		if (this.repository == null) {
+			this.repository = this.container.git.getBestRepositoryOrFirst();
+			if (this.repository == null) return undefined;
+		}
+
+		const [hiddenBranches, hiddenTags] = await Promise.all([
+			this.repository.getBranches({
+				filter: b => !b.current && hiddenRefs[b.id] != undefined,
+			}),
+			this.repository.getTags({
+				filter: t => hiddenRefs[t.id] != undefined,
+			}),
+		]);
```

Comment:
This validation has too performance impact. I would ask we not ship v13 with this validation and have it as tech debt to solve after we launch.

Reply:
@eamodio is out of the office. We discussed this yesterday and we're in agreement.

Reply:
Modified in commit: `Commented hidden refs validation for v13, as it has too much performance impact. Will be solved after we launch v13 as tech debt`.